Wangscape/tilegen/partition/TilePartitionerNoise.cpp

```diff
@@ -43,6 +44,19 @@ noise::module::ModulePtr TilePartitionerNoise::makeCornerModule(const Corners& c
     combiner.setInputModuleSource(2, central.getOutputModule());
     if (mDebugOutput)
```

Comment:
Code under this `if` should be moved to a separate function in my opinion. Notice that it's now longer than the action above, for which it saves debug info.

Hmm... Can it be moved to a separate module? Will there be any benefit? Can it be reused?